certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3294 @@
+	{
+	  "swagger": "2.0",
+	  "info": {
+		"version": "1.0.4",
+		"title": "Virus Total",
```

Comment:
It's a single word (See https://en.wikipedia.org/wiki/VirusTotal). Why are there spaces between everywhere between 'Virus' and 'Total'?

Reply:
Not a quality pull request when we cannot spell the registered service name correctly 😕. What's the reason for using 'Ip' with lower-case 'p'? Such text descriptions might take the customers off from the power platform.

Reply:
fix